src/components/grid/GridComponent.vue

```diff
@@ -132,6 +128,16 @@ export default Vue.extend({
       stickyTableHeader: false
     }
   },
+  filters: {
+    formatCount: function (value) {
+      if (value === -1) {
+        return '1-15'
```

Comment:
The 15 is a server setting, and if the customer changes their mind it may end up with a different value.
Bit unsure how to handle this.